src/App/ComponentsDocumentation/components/RadioButton/index.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { ComponentPreview, DocContainer } from "@docutils";
+import Radio from "@components/FormComponents/Radio";
+
+const ButtonExamples = () => (
+    <div>
+        <Radio id="radio-example-1" label="Phở" name="radio-example"/>
+        <Radio id="radio-example-2" label="Flied lice" name="radio-example"/>
```

Comment:
This is gonna backfire :)
```suggestion
        <Radio id="radio-example-2" label="Fried rice" name="radio-example"/>
```